### PR TITLE
fix vagrant bug and update pip lockfile

### PR DESCRIPTION
**Pull request by erickyellott**

### CODE REVIEW

**Comment by svsaraf** [comment]
Looks great!

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
-Django==2.1
+Django==2.1.2
 feedparser==5.2.1
 pytz==2018.5
+## The following requirements were added by pip freeze:
```
Comment (erickyellott) [inline requirements.txt:4]: the whole `requirements-to-freeze.txt` thing is from this guy: https://www.kennethreitz.org/essays/a-better-pip-workflow

basically put unpinned top level dependencies in `requirements-to-freeze.txt` (to make upgrades a lot easier), and use `requirements.txt` as an explicit and exhaustive lockfile.

**Comment by svsaraf** [comment]
That's actually really smart. I've read some of Reitz's posts before but
hadn't seen this one.

**Comment by erickyellott** [comment]
They're currently working on a more automated version of this called `Pipenv`, but from my tinkering, it isn't quite ready yet.

**Pull request merged**